README.md

```diff
@@ -202,10 +202,14 @@ Before you start the exercises, please look at this [example](https://github.com
 
 In these exercises, you have a number of impure functions and a number of failing tests. Your task is to rewrite the functions (and not the tests!) to make sure that the tests pass and the functions are pure.
 
-* clone this repo and run `npm install`
-* run the tests using `npm test` (don't worry about the package-lock.json notice)
-* look at the passing and failing tests
-* open `exercises/exercise1.js`
-* refactor the functions to make the tests pass. It will be useful to open the tests and look at exactly what is expected.
+- clone this repo and run `npm install`
+- if you wish to use nodemon:
+  - 1. run `npm i -D nodemon` in your terminal
+  - 2. change the test scripts in your package.json file to
+       `"test": "nodemon ./test/exercise1.test.js | tap-spec"`
+- run the tests using `npm test` (don't worry about the package-lock.json notice)
```

Comment:
What package-lock.json notice?

Reply:
This was already in the readme  - I don't remember seeing a notice at the time, but I'm not too familiar with the terminal yet 🤷‍♀️ It might no longer be relevant.